test/unit/api_spec.js

```diff
@@ -1320,6 +1333,15 @@ describe("api", function () {
       await loadingTask.destroy();
     });
 
+    it("gets a destination, from /Names (NameTree) respectively /Dests dictionary", async function () {
```

Comment:
This test "only" accesses a sinlge destination in the /Dests entry of the root dictionary.

I would suggets to add another test, that access another destination which is resolved through the /Dests name tree of the /Names dictionary ("B") for completion.